classes/PodsAPI.php

```diff
@@ -6279,7 +6316,29 @@ public function load_pods( $params = null ) {
 			}
 		}
 
-		return $this->load_object_pods( $params );
+		$loaded_pods = $this->load_object_pods( $params );
+
+		if ( empty( $loaded_pods ) ) {
+			return [];
+		}
+
+
+		if ( $include_internal || ! is_array( $loaded_pods ) ) {
+			return $loaded_pods;
+		}
+
+		// Remove the internal Pods.
+		$the_pods = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 29e5df9599f8cf48c7e0d206dfc15b10</sub>